src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5117,6 +5117,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExplicitReadOnlyAttr" xml:space="preserve">
     <value>Do not use 'System.Runtime.CompilerServices.IsReadOnlyAttribute'. This is reserved for compiler usage.</value>
   </data>
+  <data name="ERR_ExplicitIsByRefLikeAttr" xml:space="preserve">
```

Comment:
I do not think this kind of error is reported often or interesting from telemetry prospective.

---
In reply to: [118795526](https://github.com/dotnet/roslyn/pull/19795#discussion_r118795526) [](ancestors = 118795526,118795162,118793105)